docs/personalization/triggers.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Triggers enable sending recommendations as emails to customers.
+---
+
+# Email triggers
+
+With email triggers, you can increase the engagement of your visitors and customers by delivering recommendations straight to their mailboxes.
+As a store manager, you can expect that more engaged customers translate to bigger income, improved fulfillment of customer needs, while saving time and effort.
+
+## Email trigger types
+
+There are several email triggers available in [[= product_name =]], including the following two:
+
+- Abandoned basket trigger: Personalization engine monitors the user's cart and pushes a message when cart status remains unchanged for a set time. The message contains items that have been abandoned in the cart.
+
+- Reactivation aka. "We miss you" trigger: Personalization engine monitors the user's overall activity and pushes a message when they have not returned to the site for a set time. Recommendations come from the [also purchased and also clicked](recommendation_models.md#also-clicked--purchased) models, while the [top purchased](recommendation_models.md#popularity-models) model is used as fallback.
+
+## Trigger calculation frequency
+
+Personalization engine checks user context data against trigger conditions every night.
+Email triggers are automatically initiated when a specific user's action, inaction or pattern of actions meet certain conditions that are defined in the service.
+
+Depending on the trigger type and specific settings made for your organization, each email trigger might be initiated several times at set intervals.
```

Comment:
Interval is not handled yet. I think we have more important features right now (attributes/messages limit), thus I am not sure if will be able to finish it before release. What is more I think we will allow minimum interval 1h - but as I said we have not defined anything in here yet. @dkorostynski what do you think?